modules/statics/src/coins.ts

```diff
@@ -10124,6 +10124,7 @@ export const coins = CoinMap.fromCoins([
   ofcerc20('67fb4a6f-ba00-41fd-972d-728d2226a3d5', 'ofccho', 'Choise.com', 18, UnderlyingAsset.CHO),
   ofcerc20('af198c54-53d5-42b3-9e69-7fcc5887c0a0', 'ofcdia', 'DIA Token', 18, UnderlyingAsset.DIA),
   ofcerc20('4da7f6a6-9c56-44f7-a9b2-07ae7c669e42', 'ofcldo', 'Lido DAO Token', 18, UnderlyingAsset.LDO),
+  ofcerc20('4da7f6a6-9c56-44f7-a9b2-07ae7c669e42', 'ofcsbc', 'Sustainable Bitcoin Certificate', 8, UnderlyingAsset.SBC),
```

Comment:
The uuid is same as the uuid for `ofcldo` ^

Reply:
Yeah, missed